test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py

```diff
@@ -31,19 +31,35 @@
 
 
 def launch_setup(context, *args, **kwargs):
-    autoware_launch_file = LaunchConfiguration(
-        "autoware-launch-file",
-        default="planning_simulator.launch.xml"
-        # use the below launch file to enable AutowareAuto instead of ArchitectureProposal
-        # "autoware-launch-file", default="autoware_auto.launch.py"
-    )
+    # Autoware Type
+    autoware_types = ["auto", "proposal"]
+    autoware_type = LaunchConfiguration("autoware_type", default="proposal")
```

Comment:
Please change the option name to `architecture-type` and the related variable name to` architecture_type`.
Also, change `proposal` and `auto` to `tier4/proposal` and` awf/auto`.

That is, the optional formats are `architecture-type:=tier4/proposal` and` architecture-type:=awf/auto`.

This change reflects the .Auto / ArchitectureProposal identification metapackage terminology currently under development within Tier IV.